tests/tests/multiple_files_tests.robot

```diff
@@ -77,10 +76,10 @@ Extract Multiple Files
     Log To Console    ${file_list}
 
     FOR    ${file}    IN    @{file_list}
-        ${test_file_path}    Catenate    SEPARATOR=    ${test_file}[path]/${file}
+        ${test_file_path}    Set Variable     ${test_file}[path]/${file}
```

Comment:
as above

Reply:
Agreed with Krzysztof that 'Set variable' topic will be handle by separate PR.